controllers/dspastatus/dspa_status.go

```diff
@@ -35,6 +38,7 @@ func NewDSPAStatus(dspa *dspav1.DataSciencePipelinesApplication) DSPAStatus {
 	persistenceAgentCondition := BuildUnknownCondition(config.PersistenceAgentReady)
 	scheduledWorkflowReadyCondition := BuildUnknownCondition(config.ScheduledWorkflowReady)
 	mlmdProxyReadyCondition := BuildUnknownCondition(config.MLMDProxyReady)
+	webhookReadyCondition := BuildUnknownCondition(config.WebhookReady)
```

Comment:
For the case where the DSPA isn't using the webhook/Kubernetes pipeline store, it'd be nice to have something other than unknown such as not applicable?

Reply:
Why not just remove that condition if Webhook is not deployed? Having a condition to an unused Deployment might be confusing to users.